src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -69,13 +74,17 @@ const GiftPopover = (
   </Popover>
 );
 
-const RadioPopover: React.FC<RadioModalPropsType> = (props) => (
-  <Popover id="radio-popover">
-    <Popover.Content>
-      <RadioModal {...props} />
-    </Popover.Content>
-  </Popover>
-);
+// TODO: Get rid of this as soon as humanly possible.
+// TODO: Look into properly setting up SoundClouds' API,
+// possibly without the need to append script
+const loadScript = (src: string) => {
```

Comment:
This is backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/62